network/endpoint.go

```diff
@@ -30,6 +30,7 @@ type AzureHNSEndpoint struct{}
 type endpoint struct {
 	Id                       string
 	HnsId                    string `json:",omitempty"`
+	HNSNetworkID             string `json:",omitempty"`
```

Comment:
can we add a comment that this is used only in windows?

Reply:
HNS is only for Windows so no need for comment